css/selectors/nesting.json

```
@@ -0,0 +1,41 @@
+{
+  "css": {
+    "selectors": {
+      "nesting": {
+        "__compat": {
+          "description": "Nesting selector (<code>&</code>)",
+          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/nesting",
```

Not convinced by this URL (and its capitalization). 

We have [Attritube_selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Attribute_selectors) and [Class_selectors](https://developer.mozilla.org/en-US/docs/Web/CSS/Class_selectors), …

We should follow the pattern:
```suggestion
          "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/Nesting_selectors",
```